pyproject.toml

```diff
@@ -27,6 +27,7 @@ GeoAlchemy2 = "^0.10.2"
 python-multipart = "^0.0.5"
 asyncpg = "^0.25.0"
 gunicorn = "^20.1"
+pytest = "^7.1.1"
```

Comment:
The docker build fails due to this. Not sure why, 7.1.1 is a valid version

Reply:
The reason it fails is because @diegoac2 needs to check in an updated poetry.lock file. It looks like other dependencies pull in lower versions of pytest and we specify 7.1.1